[ONNX] replace all _C._flatten to torch.jit._flatten

titaiwangms opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #83186
* #83201
* #83599
* __->__ #83598

_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

titaiwangms committed:
replace all _C._flatten to torch.jit._flatten

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them can reduce confusing.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them can reduce confusing.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them can reduce confusing.

[ghstack-poisoned]

justinchuby commented on test/test_jit.py:
```diff
@@ -1201,7 +1201,7 @@ def test_arg_configurations(self):
         @torch.jit.compile(nderivs=0)
         def fn(*args):
-            in_vars, _ = torch._C._jit_flatten(args)
+            in_vars, _ = torch.jit._flatten(args)
```
Looks like `ONNX exporter` doesn't have merge permission on this file. changing it can take longer

AllenTiTaiWang replied:
I can change this one back.

AllenTiTaiWang replied:
Done

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] replace all _C._flatten to torch.jit._flatten"


_C._flatten is exactly the same as torch.jit._flatten. Unifying them to reduce confusion.

[ghstack-poisoned]

AllenTiTaiWang commented:
@pytorchbot merge -g
